ext/websocket/02_websocketstream.js

```diff
@@ -265,9 +259,7 @@ class WebSocketStream {
                 }
                 case 3: {
                   /* ping */
-                  await core.opAsync("op_ws_send", this[_rid], {
-                    kind: "pong",
-                  });
+                  await core.opAsync("op_ws_send_pong", this[_rid]);
```

Comment:
I dont think this pong response is necessary; or at least, we dont send it in the normal websocket impl

Reply:
@littledivy suggested that it is necessary

Reply:
this isn't necessary. pong frames are sent automatically but not pong frames (in timeouts)